hadoop-ozone/integration-test/src/test/resources/core-site.xml

```diff
@@ -20,5 +20,16 @@
 <!-- Put site-specific property overrides in this file. -->
 
 <configuration>
-
+  <property>
+    <name>hadoop.proxyuser.proxyuser.users</name>
+    <value>*</value>
+  </property>
+  <property>
+    <name>hadoop.proxyuser.proxyuser.groups</name>
+    <value>*</value>
+  </property>
+  <property>
+    <name>hadoop.proxyuser.proxyuser.hosts</name>
+    <value>*</value>
+  </property>
 </configuration>
```

Comment:
Is this required for new added tests?

Reply:
This is used for tests. Because I found that adding proxyuser to the specific test case doesn't work.